lib/ansible/module_utils/api.py

```diff
@@ -114,3 +115,55 @@ def retried(*args, **kwargs):
 
         return retried
     return wrapper
+
+
+def generate_jittered_backoff(retries=10, delay=3, max_delay=60):
```

Comment:
Could `delay` be `delay_base`? The explanation below says that it's initial but the reality is that it's used on each iteration, not just the first one...

Also `max_delay` sounds more like a `delay_threshold` maybe?

WDYT?

Reply:
renamed those :+1: